prod_api/content_types/service.py

```diff
@@ -16,9 +16,11 @@
 
 class ContentTypesService(superdesk.Service):
     def get_output_name(self, profile):
+        service = superdesk.get_resource_service("content_types")
         try:
             _id = bson.ObjectId(profile)
-            item = superdesk.get_resource_service("content_types").find_one(req=None, _id=_id) or {}
-            return format_content_type_name(item, str(_id))
+            item = service.find_one(req=None, _id=_id) or {}
```

Comment:
using `item` here is a bit confusing, profile would be a better name probably, and the param could be `profile_id`